db/migrate/20160805000209_create_contacts.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContacts < ActiveRecord::Migration
+  def change
+    create_table :contacts do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :phone
+      t.string :service
+      t.date :date
+      t.string :availability
+      t.integer :skillset
+      t.text :notes
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
Yes, I can make that change.  Service is the social thing that you met them.  What is a better name?  

303.601.8014
www.LaurieLinz.com
Github:https://github.com/LaurieLinz
Blog:https://laurielinz.wordpress.com/
LinkedIn: https://linkedin.com/in/lauralinz
Mail: lalinz303@gmail.com
Twitter: @laurielinz





> On Aug 6, 2016, at 3:40 PM, Rylan Bowers <notifications@github.com> wrote:
> 
> In db/migrate/20160805000209_create_contacts.rb <https://github.com/LaurieLinz/organize-your-network/pull/1#discussion_r73792382>:
> 
> > +  def change
> > +    create_table :contacts do |t|
> > +      t.string :first_name
> > +      t.string :last_name
> > +      t.string :email
> > +      t.string :phone
> > +      t.string :service
> > +      t.date :date
> > +      t.string :availability
> > +      t.integer :skillset
> > +      t.text :notes
> > +
> > +      t.timestamps null: false
> > +    end
> > +  end
> > +end
> First name, Last Name, skillset, date,
> 
> Can we rename date to date_met and what is service? Should be more clear? Service or social thing I met them through?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/LaurieLinz/organize-your-network/pull/1/files/9ef65a11f5f83a5f7a0d94b04cf928ba6d4ef348#r73792382>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AOKlg2nvEqiwTGNPeNwtTYBFrufXYXgkks5qdP88gaJpZM4JdP6T>.
>